examples/interactive_oauth.py

```diff
@@ -1,4 +1,4 @@
-from databricks import sql
+from databricks_sql_connector_core import sql
```

Comment:
all the examples should still work w/o any code change, we should not change these. If it is required, it means this change will break all the customer codes.